demo/c/README.md

```diff
@@ -28,16 +37,16 @@ Running the executable without any commandline arguments prints the usage info t
 
 ```console
 ./demo/c/build/porcupine_demo_mic 
-usage : ./demo/c/build/porcupine_demo_mic library_path model_path keyword_path sensitivity audio_device_intex
-        ./demo/c/build/porcupine_demo_mic --show_audio_devices
+Usage : ./demo/c/build/porcupine_demo_mic -l LIBRARY_PATH -m MODEL_PATH -k KEYWORD_PATH -t SENSTIVITY -a ACCESS_KEY -d AUDIO_DEVICE_INDEX
+        ./demo/c/build/porcupine_demo_mic [-s] [--show_audio_devices]
```

Comment:
`[-s, --show_audio_devices]` is the style that `man` is using i guess if they both mean the same thing